app/controllers/user/attendances_controller.rb

```diff
@@ -42,4 +42,12 @@ def update_zoom_alias
     attendance.rerecord
     redirect_to attendance_path(attendance)
   end
+
+  def destroy
+    attendance = Attendance.find(params[:id])
+    module_id = attendance.turing_module.id
+    attendance.destroy
+    redirect_to turing_module_path(module_id)
+    flash[:sucess] = "Attendance successfully deleted."
```

Comment:
Would it make sense to log what `attendance` is before the destroy so that there is at least an audit log of what got deleted? Probably `INFO` level would be good enough.

Reply:
Thanks for pointing this out. We created a new user story on the project board to ensure that this does not get missed. If this is a priority for you we can resolve that user story right away.

We can up with two viable solutions to do this. One is using a gem like paper trail to track deleted objects. But since there are no other objects being deleted, adding a whole gem for this specific task might be going overboard. 

The other is converting the delete action into a soft delete with an enum as an attribute. So the record would still persist, but we can make them only accessible and able to be seen by admins and not regular users.

Let us know your thoughts!

Reply:
What about just using `puts` or the Rails Logger? https://guides.rubyonrails.org/debugging_rails_applications.html#the-logger

I don't think you'd need a separate gem. Anything that gets logged to stdout gets picked up by Heroku, which I think would be good enough.

The soft delete idea is also interesting. I don't have strong opinions if it makes sense in this case, but in general, it's a good concept if you think you may want to keep the data around.

Reply:
Thank you for the insight on using the rails logger. We have updated this action to include a logger that will save the information of the deleted attendances in case it needs to be inspected in the future.